sycl/include/CL/sycl/kernel_bundle.hpp

```diff
@@ -503,7 +497,8 @@ bool has_kernel_bundle(const context &Ctx, const std::vector<device> &Devs) {
 
 /// \returns true if all of the kernels identified by KernelIDs are compatible
 /// with the device Dev.
-bool is_compatible(const std::vector<kernel_id> &KernelIDs, const device &Dev);
+__SYCL_EXPORT bool is_compatible(const std::vector<kernel_id> &KernelIDs,
```

Comment:
NIT: I see some changes like this that do not really relate to kernel bundles, should better be separate PRs